config/rollup.config.js

```diff
@@ -40,8 +38,10 @@ module.exports = {
     reactSvg(),
     copy({
       targets: [
-        { src: 'lib/styles/build/**/*.{css,js}', dest: 'dist' }
-      ]
-    })
+        { src: 'lib/styles/build/**/*.{css,js}', dest: 'dist' },
+        { src: 'lib/styles/typography/fonts/', dest: 'dist' },
+        { src: 'lib/styles/typography/css/*.css', dest: 'dist/font-style' },
```

Comment:
Maybe we can simplify the structure of `lib/styles/typography` in the near future? I was hoping for something like a **fonts.css** that we can add to `dist` for easier importing. 

This works for now but is very specific when importing elsewhere (like the example app):

```html
<link 
  rel="stylesheet" 
  href="https://unpkg.com/virtuoso-design-system@latest/dist/font-style/open-sans.css">
<link 
  rel="stylesheet" 
  href="https://unpkg.com/virtuoso-design-system@latest/dist/font-style/raleway.css">
```

Reply:
vs. a one-liner like:
```html
<link 
  rel="stylesheet" 
  href="https://unpkg.com/virtuoso-design-system@latest/dist/fonts.css">
```

Reply:
Per our convo, resolving.